cves/CVE-2004-0885.yml

```diff
@@ -77,9 +87,10 @@ unit_tested:
 
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  answer:
-  code:
-  fix:
+  answer: The vulnerability was discovered by a tester/user. Tests may have been
+    implemented to verify proper fix.
+  code: false
+  fix: true
```

Comment:
Are you sure that they added unit tests? I can't find any for this module.

Reply:
You are right, that should be false. I'll fix that right now.